<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dsSingleImg</class>
 <widget class="QWidget" name="dsSingleImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_tree">
       <property name="placeholderText">
        <string>搜索</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <column>
        <property name="text">
         <string>表号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_chart"/>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="text">
    <string>添加曲线</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
